opensoundscape/audio.py

```diff
@@ -137,8 +137,28 @@ def from_bytesio(
     def __repr__(self):
         return f"<Audio(samples={self.samples.shape}, sample_rate={self.sample_rate})>"
 
+    def resample(self, sample_rate):
+        """ Resample Audio object
+
+        Args:
+            sample_rate: the new sample rate
+
+        Returns:
+            a new Audio object of the desired sample rate
+        """
+        samples_resampled = librosa.resample(
+            self.samples, orig_sr=self.sample_rate, target_sr=sample_rate
+        )
+
+        return Audio(
+            samples_resampled,
+            sample_rate,
+            resample_type=self.resample_type,
```

Comment:
resample type refers to the algorithm for resampling. So maybe the librosa.resample call should use the audio object's self.resample_type - otherwise, including self.resample_type in the new object is somewhat misleading. Resample-type could (should?) also be an optional argument to .resample()

Reply:
Added a `resample_type` argument that by default is `self.resample_type` of the original `Audio` instance. The `resample_type` used in this method is used to initialize the new Audio object.